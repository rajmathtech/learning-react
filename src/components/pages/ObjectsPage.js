import React from 'react';
import Objects from '../Objects';
const ObjectsPage = () => (<div className="container__main"> 
    <h1 className="heading-primary"> Welcome to Objects list </h1> 
    <Objects />
</div>);

export default ObjectsPage;