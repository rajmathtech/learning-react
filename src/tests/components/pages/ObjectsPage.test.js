import React from 'react';
import {shallow} from 'enzyme';
import ObjectsPage from '../../../components/pages/ObjectsPage';
test('should render objects component', () => {
    const wrapper = shallow(<ObjectsPage />);
    expect(wrapper).toMatchSnapshot();   
});