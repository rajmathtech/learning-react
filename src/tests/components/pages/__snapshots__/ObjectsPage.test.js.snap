// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render objects component 1`] = `
<div
  className="container__main"
>
  <h1
    className="heading-primary"
  >
     Welcome to Objects list 
  </h1>
  <Connect(Objects) />
</div>
`;
